microsoft-edge/devtools-guide-chromium/customize/theme.md

```diff
@@ -0,0 +1,98 @@
+---
+description: How to apply different color themes to Microsoft Edge DevTools.
+title: Apply color themes to Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/22/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+<!-- Copyright Kayce Basques 
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       https://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.  -->
+# Apply color themes to Microsoft Edge DevTools  
+
+You can apply various color themes to Microsoft Edge DevTools, including several themes from [Visual Studio Code][VSCode], such as Monokai and Solarized Dark.  Themes affect the color of panes, buttons, and code syntax highlighting. 
+
+:::image type="complex" source="./media/all-devtools-themes.png" alt-text="Various DevTools color themes" lightbox="./media/all-devtools-themes.png":::
```

Comment:
Needs transparent background.  I'm trying to reuse this .png file in What's New 93 and my PR has a review comment that requests transparent background.  I tried Paint 3D > **Canvas > Transparent canvas**; didn't work.